ui/app/templates/components/database-role-edit.hbs

```diff
@@ -124,43 +126,38 @@
         @dbType={{await this.databaseType}}
       />
 
+      {{#if (not (is-empty-value this.warningMessages))}}
```

Comment:
Disabling form submit is an antipattern, and info in tooltips is not accessible. Updated to:
<img width="1440" alt="Screenshot 2023-10-17 at 1 52 32 PM" src="https://github.com/hashicorp/vault/assets/82459713/9aec0ac9-e133-40df-b265-c9b3261ea021">

Reply:
It seems like an `AlertInline` beside the submit buttons aligns more with our existing form pattern 

(Eventually that component will be replaced with `<Hds::Alert @type="compact">` unless we decide on a new pattern all together with forms)

Reply:
Yeah - this has just been our pattern so this is what I was thinking
<img width="440" alt="Screenshot 2023-10-18 at 10 21 32 AM" 
src="https://github.com/hashicorp/vault/assets/68122737/52382987-8930-47b3-ab29-499ddceec048">

Reply:
There's some weird spacing stuff going on with this form so I'll need to refactor it to get it to match.

Reply:
I'll defer to whatever you think! Was just an observation :)